app/src/main/java/org/oppia/app/profileprogress/ProfileProgressActivityPresenter.kt

```diff
@@ -21,4 +34,25 @@ class ProfileProgressActivityPresenter @Inject constructor(private val activity:
   private fun getProfileProgressFragment(): ProfileProgressFragment? {
     return activity.supportFragmentManager.findFragmentById(R.id.profile_progress_fragment_placeholder) as ProfileProgressFragment?
   }
+
+  fun showPictureEditDialog() {
+    val previousFragment =
+      activity.supportFragmentManager.findFragmentByTag(TAG_PROFILE_PICTURE_EDIT_DIALOG)
+    if (previousFragment != null) {
+      activity.supportFragmentManager.beginTransaction().remove(previousFragment).commitNow()
+    }
+    val dialogFragment = ProfilePictureEditDialogFragment.newInstance()
+    dialogFragment.showNow(activity.supportFragmentManager, TAG_PROFILE_PICTURE_EDIT_DIALOG)
+  }
+
+  fun openGalleryIntent() {
+    val galleryIntent = Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI)
+    activity.startActivityForResult(galleryIntent, GALLERY_INTENT_RESULT_CODE)
+  }
+
+  fun handleOnActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
```

Comment:
You are not using `requestCode: Int, resultCode: Int,` so remove them from this function.